wizard/cypress/e2e/uploadCsv.cy.js

```diff
@@ -3,17 +3,22 @@
 import 'cypress-file-upload';
 
 describe('Wizard from local CSV', () => {
-  it('should display the form: Metadata > Files', () => {
+  it('should display the form: Metadata, Files, & Record Sets', () => {
+    const resizeObserverLoopErrRe = /ResizeObserver loop limit exceeded/
+
+    // consensus was that this exception didn't matter mostly, and was intermittent when running tests.
+    Cypress.on('uncaught:exception', err => {
```

Comment:
Where does this exception come from in the Streamlit code?

What do you mean by "consensus"? Do you have a link to a GitHub issue?